SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -148,6 +141,12 @@ void USpatialReceiver::OnAddComponent(Worker_AddComponentOp& Op)
  		GlobalStateManager->ApplyStartupActorManagerData(Op.data);
 		return;
 	default:
+		if (!bInCriticalSection)
```

Comment:
This had to move because when a client updates its component interest upon gaining ownership over something, it will receive an `AddComponentOp` for the client endpoint outside a critical section.